application/controllers/state.py

```diff
@@ -3,13 +3,15 @@
 from flask import jsonify
 
 
-def print_json (players, cards):
+def print_json (player, players, cards):
     """print({
-        'player': [player.serialise() for player in players],
+        'player': player.serialise(),
+        'allPlayers': [player.serialise() for player in players],
         'cards': [card.serialise() for card in cards],
     })"""
     return {
-        'player': [player.serialise() for player in players],
+        'player': player.serialise(),
+        'allPlayers': [player.serialise() for player in players],
```

Comment:
This will send duplicate information. I would recommend changing lines 9 and 14 to:

'allPlayers': [p.serialise() for p in players if p is not player]

I'm not sure if this will break anything front end though so just leaving it as a comment for now.

Reply:
Yes good plan will change